content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,48 +1,105 @@
-======================================
-Allow customers to close their tickets
+===============
+Closing Tickets
+===============
+
+Once work has been completed on a :guilabel:`Helpdesk` ticket, there are several ways it can be
+closed. Manually closing solved tickets keeps the pipeline up to date, while automatically closing
+inactive tickets prevents unnecessary blocking issues. Allowing customers to close their own tickets
+minimizes confusion around whether an issue is considered solved or not. This results in increased
+operational capacity for support teams, and higher customer satisfaction.
+
+Manually closing solved tickets
+===============================
+
+As work on a ticket progresses, it is moved along to the next stage in the pipeline. Once the issue
+is solved, the ticket is moved to a *folded* stage, to signify the work has been completed. This
+marks the ticket as *closed*.
+
+To fold a stage, navigate to the :guilabel:`Helpdesk` dashboard and click on a team to open the
+pipeline. Choose a stage, and click on the gear icon at the top right of that stage's kanban column.
+
+.. image:: close_tickets/closing-edit-stage-gear.png
+   :align: center
+   :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.
+
+.. warning::
+   Clicking the gear icon also displays the option to :guilabel:`Fold` the stage. This setting folds
+   the stage **temporarily** to simplify the kanban view. This does **not** close the tickets in
+   this stage. It also does not permanently fold the stage. If a stage needs to be folded so the
+   tickets can be marked as closed, continue following the steps below.
+
+Select :guilabel:`Edit Stage`. This will open the stage's settings page. Check the box labeled
+:guilabel:`Folded in Kanban`. :guilabel:`Save & Close`.
+
+   .. image:: close_tickets/closing-folded-setting.png
+      :align: center
+      :alt: Stage settings page.
+
+Automatically closing inactive tickets
 ======================================
 
-Allowing customers to close their own tickets gives them autonomy and minimizes misunderstandings
-around when an issue is considered solved or not. This results in operational capacity for support
-teams, and higher satisfaction for the customer.
+Tickets that are inactive for a set period of time can be automatically closed. At that point,
+they will be moved to a folded stage.
 
-Enable ticket closing
-=====================
+Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
+Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
```

Comment:
```suggestion
Go to the team's :menuselection:`settings` page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
Under the :menuselection:`Self-Service` section, enable :guilabel:`Automatic Closing`.
```

Reply:
The first way is correct.
Another way to write this so it's a compound sentence with time-order language.
```suggestion
Enable automatic ticket closing by first navigating to :menuselection:`Helpdesk --> Configuration
--> Teams`, and then under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic
Closing`.
```